.github/workflows/build_test_deploy.yml

```diff
@@ -2,7 +2,7 @@ name: Test, build, and deploy
 
 on:
   push:
-    branches: [ main ]
+    branches: [ main , fix/ci-issue ]
```

Comment:
Do we want both branches here?

Reply:
Oh sorry I thought I'd taken that out. It was so I could test it.

Reply:
Should be gone now.